src/plugins/exec/sim-host-dialogs.html

```diff
@@ -4,8 +4,7 @@
     <cordova-group>
         <p>There is no handler for the following exec call:</p>
         <p style="font-style:italic;color:black"><span id="exec-service">service</span>.<span id="exec-action">action</span></p>
-        <p>Please enter a value below to be returned for this call, then indicate whether this represents success or failure.
-            If you wish to always respond to this call with this result, select the 'persist' option.</p>
+        <p>Please enter a value below to be returned for this call, then indicate whether this represents success or failure. If you wish to always respond to this call with this result, select the 'persist' option.</p>
```

Comment:
Why the change here? Was this an autoformat change, or are there restrictions on how newlines interact with loc?